0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -191,106 +142,23 @@ Content-Type: application/json
 }
 ```
 
-##### Invoice
-
-Invoice information includes an exact amount as well as the status of the invoice. (Invoices can only be paid once.)
-
-Example Receiver:
-``` json
-{
-  "type": "invoice",
-  "account": "ilpdemo.red.amazon.111-7777777-1111111",
-  "currency_code": "USD",
-  "currency_symbol": "$",
-  "amount": "10.40",
-  "status": "unpaid",
-  "invoice_info": "https://www.amazon.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111"
-}
-```
-
-| Field | Type | Description |
-|---|---|---|
-| `type` | `"invoice"` | Receiver type |
-| `account` | ILP Address | ILP Address of the recipient's account |
-| `currency_code` | String | Currency code to identify the receiver's currency. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. Sender UIs SHOULD be able to render non-standard codes |
-| `currency_symbol` | String | Symbol for the receiver's currency intended for display in the sender's UI (e.g. `"$"` or `"shares"`). Sender UIs SHOULD be able to render non-standard symbols |
-| `amount` | Decimal String | Value of the invoice in the recipient's currency |
-| `status` | Enum: `"paid"`, `"unpaid"`, `"cancelled"` | State of the invoice |
-| `invoice_info` | URI | URI where additional information about the invoice can be found. |
-
-### Setup (`POST <receiver>`)
-
-When the sender is ready to make a payment, it submits a payment object to the receiver.
-
-#### Request
-
-##### For a Payee
-
-``` http
-POST /api/receiver/bob HTTP/1.1
-Host: red.ilpdemo.org
-Accept: application/json
-
-{
-  "amount": "10.40",
-  "sender_identifier": "alice@blue.ilpdemo.org",
-  "memo": "Hey Bob!"
-}
-```
-
-| Field | Type | Description |
-|---|---|---|
-| `amount` | Decimal String | _Destination amount_ the receiver will receive |
-| `sender_identifier` | String | Identifier of the sender |
-| `memo` | String | Message for the recipient linked to the payment |
+### Payment Setup
 
-##### For an Invoice
+The sender uses the receiver details to create the ILP packet:
+* The `account` in the ILP packet is the `destination_account` provided by the receiver
```

Comment:
Add a blank line between the lead-in and the start of the bulleted list. Some Markdown parsers don't recognize a list if it's not separated from a paragraph by a blank line.